dangerfile.js

```diff
@@ -113,6 +113,10 @@ function row(result) {
     return;
   }
 
+  // Disable sizeBot in a Devtools PR like bellow title. Because that doesn't affect production bundle size.
+  // [Fix DevTools console announcement being suppressed by Fast Refresh #21864](https://github.com/facebook/react/pull/21864)
+  if (new RegExp('devtools?', 'i').test(danger.github.pr.title)) return;
```

Comment:
I like the idea of this pull request, but I think maybe we shouldn't rely on the name of the title. (There may be DevTools-focused PRs that also change things in non-DevTools code.)

If we can instead look at the list of changed files and skip it if all files are in `packages/react-devtools*` then that would be okay.

Reply:
@bvaughn Thank you for your decent review!
Totally I agree with you, actually I could only achieved by this way in a short spare time 😅

> look at the list of changed files and skip it if all files are in packages/react-devtools* then that would be okay.

I'll follow and implement this safety way and then I'll mention you review request again.
To making **detect only `packages/react-devtools*` files** logic I wanna little bit explore codebase for seek something effective util code in this situation.
So might be I'll update my work within about a week.


**misc.** I searched about toplevel **return** statement in node.js and stackoverflow says that equivalent to `process.exit()`.
I learned `return` `process.exit(0)` `process.exit()` are all equivalent exit status today 🤗

Reply:
> I'll follow and implement this safety way and then I'll mention you review request again.
To making **detect only `packages/react-devtools*` files** logic I wanna little bit explore codebase for seek something effective util code in this situation.
So might be I'll update my work within about a week.

Sounds good 👍🏼